deepmd/entrypoints/train.py

```diff
@@ -505,9 +505,13 @@ def update_one_sel(jdata, descriptor, one_type: bool = False):
 
 
 def update_sel(jdata):
-    log.info(
-        "Calculate neighbor statistics... (add --skip-neighbor-stat to skip this step)"
-    )
-    jdata_cpy = jdata.copy()
-    jdata_cpy["model"] = Model.update_sel(jdata, jdata["model"])
+    if jdata["model"]["descriptor"]["type"] == "se_a_mask":
```

Comment:
The conditions have been moved into the specific component classes in #2901.

Reply:
Update in latest commit.